framework/platform/bpl/include/bpl/bpl_cfg.h

```diff
@@ -51,6 +51,13 @@ namespace bpl {
 #define BPL_OPER_MODE_WDS_REPEATER 3
 #define BPL_OPER_MODE_L2NAT_CLIENT 4
 
+/* Platform Management Mode */
+#define BPL_MGMT_MODE_MULTIAP_CONTROLLER_AGENT 0 /* EasyMesh controller and agent */
+#define BPL_MGMT_MODE_MULTIAP_CONTROLLER 1       /* EasyMesh controller */
+#define BPL_MGMT_MODE_MULTIAP_AGENT 2            /* EasyMesh agent */
+#define BPL_MGMT_MODE_NOT_MULTIAP                                                                  \
```

Comment:
It seems weird to mee to put it in the same category of what runs on the platform because basically, the controller and the agent do run on this configuration so this is confusing a little bit.
I think it will be more suitable to give it a new category (passive mode?). But that just me, let see what others will say...

Reply:
It has to be in the same category, AKA Management mode, I don't want to introduce yet another category just for this.